torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -595,6 +604,11 @@ RegisterOperators reg2({
     Operator("aten::len(float[] a) -> int", listLen<Shared<DoubleList>>),
     Operator("aten::len(Tensor[] a) -> int", listLen<Shared<TensorList>>),
 
+    Operator("aten::list(int[] a) -> int[]", listList<Shared<IntList>>),
+    Operator("aten::list(float[] a) -> float[]", listList<Shared<DoubleList>>),
+    Operator("aten::list(Tensor[] a) -> Tensor[]", listList<Shared<TensorList>>),
```

Comment:
This is confusing. I thought the purpose of this constructor is to be able to cast tuples to lists. Right now it's completely useless.